src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,79 @@
+.contacts {
+  &__content {
+    @include on-tablet {
+      display: flex;
+      gap: 50px;
+    }
+
+    @include on-desktop {
+      display: flex;
+      gap: 50px;
+    }
+  }
+
+  &__info {
+    width: 100%;
+    gap: 50px;
+
+    @include on-tablet {
+      display: block;
+    }
+
+    @include on-desktop {
+      display: flex;
+      gap: 50px;
+    }
+  }
+  &__form {
+    width: 100%;
+  }
+
+  &__link {
+    text-decoration: none;
+    font-weight: 700;
+    color: $main-text-color;
+
+    &:hover {
+      color: $accent-bg-color;
+    }
+  }
+
+  &__label {
+    font-weight: 400;
+  }
+}
+
+.form {
```

Comment:
Your form inputs are missing hover and focus styles:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/222a2b82-e6f0-4a4d-b81e-98a964534009)

Reply:
Done